Week2/homework/maartjes-work.js

```diff
@@ -45,17 +45,33 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
+function computeTasksInHours(maartjesTasksX, celling) {
+  const marrtjeTasksInHours = maartjesTasksX
+    .map(task => task.duration / 60)
+    .filter(task => task >= celling);
+  return marrtjeTasksInHours;
+}
+
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const Taskhours = computeTasksInHours(tasks, 2);
```

Comment:
Variable, function and parameter names should be in camelCase (start with a lowercase letter). See [Naming conventions)(https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md).